.gitignore

```diff
@@ -0,0 +1,6 @@
+# Cloud9 IDE - http://c9.io
+.c9revisions
+.c9/
+node_modules/
+package-lock.json
+.git/
```

Comment:
There is a site to generate .gitignore very useful. I'll add something here probably :slightly_smiling_face: 
[gitignore.io](gitignore.io)